crates/virtio-queue-ser/Cargo.toml

```diff
@@ -14,5 +14,5 @@ serde = { version = "1.0.27", features = ["derive"] }
 versionize = "0.1.6"
```

Comment:
There's a typo in the commit message: vm-queue -> virtio-queue.

Reply:
Fixed